lib/tcp.ml

```diff
@@ -1,11 +1,25 @@
 module Client = struct
-  let open_connection host port =
-    let client = `TCP (`IP Ipaddr.(V4 host), `Port port) in
-    Conduit_lwt_unix.(connect ~ctx:(Lazy.force default_ctx) client)
+  let open_connection ~(env : Eio.Stdenv.t) ~sw ~(host : Ipaddr.V4.t) ~port =
+    let ip = Ipaddr.V4.to_octets host in
+    let addr = `Tcp (Eio.Net.Ipaddr.of_raw ip, port) in
+    Eio.Net.connect ~sw (Eio.Stdenv.net env) addr
   ;;
 
-  let write_bytes oc buf =
-    let buf_len = Bytes.length buf in
-    Lwt_io.write_from_exactly oc buf 0 buf_len
+  let write_bytes socket_flow buf =
+    Eio.Buf_write.with_flow socket_flow (fun buf_write ->
```

Comment:
Note: this creates a new buffered writer every time you write, which is probably not what you want!

If you don't need buffering, `Flow.copy_string` would be more efficient.

Reply:
Thank you! I did this fast just to pass the tests and forgot to edit it haha.
Fixed with https://github.com/fraidev/tornado/commit/9c3a6ab2e10441acb96cd586cd9b4ec847bbe94f